frontend/tests/e2e/testCases/studentAdmin/institute/insertSchoolPrincipal.js

```diff
@@ -0,0 +1,12 @@
+const log = require('npmlog');
+const { setupSchoolPrincipal } = require('../../../helpers/schoolUtils');
+
+setupSchoolPrincipal().then(async (data) => {
```

Comment:
I feel like this requires that the school 00599999 - EDX Team School exists already? I think we are supposed to delete the school at the end of test to wipe the slate clean. 

I think we can leverage your method that adds a school with the contact.

Reply:
Resolved, we are keeping a testSchool for the time being.